chain/chunks/Cargo.toml

```diff
@@ -33,3 +34,4 @@ assert_matches = "1.5.0"
 [features]
 byzantine_asserts = ["near-chain/byzantine_asserts"]
 expensive_tests = []
+test_features = []
```

Comment:
comment. Also maybe we should call it 'reconstruct_partial_chunk_on_the_fly' or something like that?

Reply:
I’m planning to make GCing a configurable feature.